planex/mock.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+
+"""
+planex-mock: Wrapper around mock
```

Comment:
Please call this `planex-build-mock`.   Then we can have `planex-build-copr`, `planex-build-docker`  etc etc etc.